### PR TITLE
Opgops 1910

### PR DESCRIPTION
**Pull request by brettminnie**
* Refactored RDS creation into own task
 * RDS SG creation happens earlier in stack creation to avoid timing issues
 * Can create ad-hoc users from meta-data now

### CODE REVIEW

File: roles/awscommon/tasks/main.yml [hunk 1]
```diff
@@ -16,3 +16,35 @@
...
+  add_host:
+    name: "jump.{{ vpc_name }}.{{ opg_data.domain }}"
+    groups: "vpc, jumphost"
+    hostname: "jump.{{ vpc_name }}.{{ opg_data.domain }}"
+    ansible_user: "{{ ssh_prov_user|default('jenkins-agent') }}"
+    ansible_ssh_common_args: "{{ ssh_conn_opts| default(omit) }}"
+  tags:
+    - always
+
+- name: Add instances to runtime group for later usage
```
Comment (tim-mcarthur-moj) [inline roles/awscommon/tasks/main.yml]: this doesn't provide us with logical groupings by function which allows per group vars

File: roles/awscommon/tasks/main.yml [hunk 2]
```diff
@@ -16,3 +16,57 @@
...
+    hostname: "{{ item.private_ip_address }}"
+    ansible_user: "{{ ssh_prov_user|default('jenkins-agent') }}"
+    ansible_ssh_common_args: "{{ ssh_conn_opts| default(omit) }}"
+    create_time: "{{ item.launch_time | default(omit) }}"
+  with_items: "{{ vpc_instance_facts.instances | default([]) }}"
+  no_log: true
+  tags:
+    - always
+
+- name: Add target level instances to runtime group for later usage
```
Comment (tim-mcarthur-moj) [inline roles/awscommon/tasks/main.yml:61]: this breaks on existing environments where the role tag is missing from jump and master

**Review by tim-mcarthur-moj — CHANGES_REQUESTED** [review]

File: roles/awscommon/tasks/main.yml (hunk 2 above)
Comment (brettminnie) [reply roles/awscommon/tasks/main.yml:61]: @tim-mcarthur-moj addressed, defaulted to `unknown` if missing

**Review by tim-mcarthur-moj — APPROVED** [review]

**Comment by tim-mcarthur-moj** [comment]
tested on
https://jenkins.service.opg.digital/job/OPG/job/base_ansible_runner/
job numbers: 1435,1436,1437

**Pull request merged**